Mathlib/CategoryTheory/Bicategory/NaturalTransformation/Oplax.lean

```diff
@@ -6,22 +6,47 @@ Authors: Yuma Mizuno, Calle Sönne
 import Mathlib.CategoryTheory.Bicategory.Functor.Oplax
 
 /-!
-# Oplax transformations between oplax functors
+# Transformations between oplax functors
 
-Just as there are natural transformations between functors, there are oplax transformations
-between oplax functors. The equality in the naturality of natural transformations is replaced by a
+Just as there are natural transformations between functors, there are transformations
+between oplax functors. The equality in the naturality condition of a natural transformation gets
+replaced by a specified 2-morphism. Now, there are three possible types of transformations (between
+oplax functors):
+* Oplax natural transformations;
+* Lax natural transformations;
+* Strong natural transformations.
+These differ in the direction (and invertibility) of the 2-morphisms involved in the naturality
+condition.
+
+In this file we define both oplax and strong natural transformations.
+
+
+The equality in the naturality of natural transformations is replaced by a
 specified 2-morphism `F.map f ≫ app b ⟶ app a ≫ G.map f` in the case of oplax natural
 transformations.
```

Comment:
This paragraph feels like it's duplicating what you wrote above. Did you mean to either delete it or incorporate it in the above?

Reply:
Oh yes my bad, thanks!